docs/NOW-spec.md

```diff
@@ -108,6 +117,20 @@ packet-beta
 
 **str (variable)**: The UTF-8 encoded string excluding the null terminator.
 
+##### NOW_GUID
+
+The NOW_GUID structure is a GUID encoded in a NOW_VARSTR structure, as a lowercase string of 36 characters. Implementations should use the NOW_VARSTR type, but refer to the NOW_GUID definition for validation.
+
+```mermaid
+packet-beta
+  0-31: "len (variable)"
```

Comment:
Let's use `NOW_VARU32` for length here; we already use it in `NOW_VARSTR`/`NOW_VARBUF`

Also, I just found that `NOW_VARBUF` is missing from a specification, but it is exactly the same type internally -- variable-length byte buffer. Could you please add it

Reply:
Ah, I read it wrong 🤦 , I missed this part - `Implementations should use the NOW_VARSTR type, but refer to the NOW_GUID definition for validation`. Never mind, everything is fine